common/changes/@microsoft/resolve-chunk-plugin/octogonz-ae-newlines-fix_2019-11-20-04-21.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/resolve-chunk-plugin",
+      "comment": "Remove `@types/webpack` from devDependencies",
```

Comment:
Weren't these removed from `dependencies` to devDeps?

Reply:
After chatting with @iclanton I converted them to peer dependencies